charts/ceph-osd-exporter/templates/clusterrole.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if and .Values.rbac.create .Values.rbac.clusterscoped }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "ceph_osd_exporter.fullname" . }}-clusterrole
+  labels:
+    {{- include "ceph_osd_exporter.labels" . | nindent 4 }}
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["list"]
+{{- end }}
```

Comment:
why do we need this role?
i think this exporter just consumes the file-based socket with hostpath mount.

Reply:
IIRC, we need this permission, but kinda forget why at this point. Let me rebult an environment and get the information back again.

Reply:
The env seem working without the permission, let me remove it now